### PR TITLE
[zh-cn] Translate API reference daemonset-v1

### PR DESCRIPTION
**Pull request by sarazqy**
translate content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md
into Chinese

Fixes https://github.com/kubernetes/website/issues/34733

### CODE REVIEW

**Comment by windsonsea** [comment]
Need wait for a dependency to merge. FYI:
```
ERROR 2022/07/13 12:29:44 [zh-cn] REF_NOT_FOUND: Ref "../workload-resources/pod-template-v1": 
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md [hunk 1]
```diff
@@ -0,0 +1,1292 @@
...
+<hr>
+
+<!--
+- **selector** (<a href="{{< ref "../common-definitions/label-selector#LabelSelector" >}}">LabelSelector</a>), required
+
+  A label query over pods that are managed by the daemon set. Must match in order to be controlled. It must match the pod template's labels. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors
+-->
+- **selector** (<a href="{{< ref "../common-definitions/label-selector#LabelSelector" >}}">LabelSelector</a>), 必须
+
+  对由守护程序集管理的 Pod 的标签查询。必须匹配才能被控制。它必须与 Pod 模板的标签匹配。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md:92]:
```suggestion
  对由守护程序集管理的 Pod 的标签查询。Pod 必须匹配此查询才能被此 DaemonSet 控制。查询条件必须与 Pod 模板的标签匹配。
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md [hunk 2]
```diff
@@ -0,0 +1,1292 @@
...
+  更多信息： https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md:93]:
```suggestion
  更多信息： https://kubernetes.io/zh-cn/docs/concepts/overview/working-with-objects/labels/#label-selectors
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md [hunk 3]
```diff
@@ -0,0 +1,1292 @@
...
+
+<!--
+- **template** (<a href="{{< ref "../workload-resources/pod-template-v1#PodTemplateSpec" >}}">PodTemplateSpec</a>), required
+
+  An object that describes the pod that will be created. The DaemonSet will create exactly one copy of this pod on every node that matches the template's node selector (or on every node if no node selector is specified). More info: https://kubernetes.io/docs/concepts/workloads/controllers/replicationcontroller#pod-template
+-->
+- **template** (<a href="{{< ref "../workload-resources/pod-template-v1#PodTemplateSpec" >}}">PodTemplateSpec</a>), 必须
+
+  描述将要创建的 Pod 的对象。DaemonSet 将在与模板的节点选择器匹配的每个节点上（如果未指定节点选择器，则在每个节点上）准确创建此 Pod 的副本。
+  更多信息： https://kubernetes.io/docs/concepts/workloads/controllers/replicationcontroller#pod-template
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md:103]:
```suggestion
  更多信息： https://kubernetes.io/zh-cn/docs/concepts/workloads/controllers/replicationcontroller#pod-template
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md [hunk 4]
```diff
@@ -0,0 +1,1292 @@
...
+
+<!--
+- **minReadySeconds** (int32)
+
+  The minimum number of seconds for which a newly created DaemonSet pod should be ready without any of its container crashing, for it to be considered available. Defaults to 0 (pod will be considered available as soon as it is ready).
+-->
+- **minReadySeconds** (int32)
+
+  新建的 DaemonSet Pod 应该在没有任何容器崩溃的情况下准备好的最小秒数，这样它才会被认为是可用的。默认为 0（Pod 准备就绪后将被视为可用）。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md:112]:
```suggestion
  新建的 DaemonSet Pod 应该在没有任何容器崩溃的情况下处于就绪状态的最小秒数，这样它才会被认为是可用的。
  默认为 0（Pod 准备就绪后将被视为可用）。
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md [hunk 5]
```diff
@@ -0,0 +1,1292 @@
...
+  -->
+  <a name="DaemonSetUpdateStrategy"></a>
+  **DaemonSetUpdateStrategy 是一个结构体，用于控制 DaemonSet 的更新策略。**
+
+  <!--
+  - **updateStrategy.type** (string)
+
+    Type of daemon set update. Can be "RollingUpdate" or "OnDelete". Default is RollingUpdate.
+  -->
+  - **updateStrategy.type** (string)
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md:135]: 135 前面空行

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md [hunk 6]
```diff
@@ -0,0 +1,1292 @@
...
+
+  守护程序集更新的类型。可以是 "RollingUpdate" 或 "OnDelete"。默认为 RollingUpdate。
+
+  <!-- 
+  - **updateStrategy.rollingUpdate** (RollingUpdateDaemonSet)
+
+    Rolling update config params. Present only if type = "RollingUpdate".
+  -->
+  - **updateStrategy.rollingUpdate** (RollingUpdateDaemonSet)
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md:144]: 144 前面添加空行

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md [hunk 7]
```diff
@@ -0,0 +1,1292 @@
...
+    -->
+    - **updateStrategy.rollingUpdate.maxSurge** (IntOrString)
+
+    在更新期间可以拥有更新的 DaemonSet Pod 的现有可用 DaemonSet Pod 的最大节点数。值可以是绝对数量（例如：5）或所需 Pod 的百分比（例如：10%）。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md:162]:
```suggestion
    对于拥有可用 DaemonSet Pod 的节点而言，在更新期间可以拥有更新后的 DaemonSet Pod 的最大节点数。
    属性值可以是绝对数量（例如：5）或所需 Pod 的百分比（例如：10%）。
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md [hunk 8]
```diff
@@ -0,0 +1,1292 @@
...
+    如果 MaxUnavailable 为 0，则该值不能为 0。绝对数是通过四舍五入从百分比计算得出的，最小值为 1。
+    默认值为 0。示例：当设置为 30% 时，最多为节点总数的 30%应该运行守护程序 Pod（即 status.desiredNumberScheduled）
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md:164]:
```suggestion
    默认值为 0。示例：当设置为 30% 时，最多为节点总数的 30% 节点上应该运行守护程序 Pod（即 status.desiredNumberScheduled）
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md [hunk 9]
```diff
@@ -0,0 +1,1292 @@
...
+    可以在旧 Pod 标记为已删除之前创建一个新 Pod。更新首先在 30% 的节点上启动新的 Pod。
+    一旦更新的 Pod 可用（至少准备好 minReadySeconds），该节点上的旧 DaemonSet pod 就会被标记为已删除。
+    如果旧 Pod 因任何原因变得不可用（Ready 转换为 false、被驱逐或耗尽），则会立即在该节点上创建更新的 Pod，而不考虑激增限制。
+    允许激增意味着如果准备检查失败，任何给定节点上的守护程序集消耗的资源可能会翻倍，因此资源密集型守护程序集应该考虑到它们可能会在中断期间导致驱逐。
+    这是 beta 字段，由 DaemonSetUpdateSurge 功能门启用/禁用。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md:169]:
```suggestion
    此字段是 Beta 字段，由 DaemonSetUpdateSurge 特性门控启用/禁用。
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md [hunk 10]
```diff
@@ -0,0 +1,1292 @@
...
+  
+      <!-- 
+      <a name="IntOrString"></a>
+      *IntOrString is a type that can hold an int32 or a string.  When used in JSON or YAML marshalling and unmarshalling, it produces or consumes the inner type.  This allows you to have, for example, a JSON field that can accept a name or number.*
+      -->
+      <a name="IntOrString"></a>
+      **IntOrString 是一种可以容纳 int32 或字符串的类型。在 JSON 或 YAML 编组和解组中使用时，它会生成或使用内部类型。例如，这允许你拥有一个可以接受名称或数字的 JSON 字段。**
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md:176]:
```suggestion
      **IntOrString 是一种可以容纳 int32 或字符串的类型。在 JSON 或 YAML 编组和解组中使用时，它会生成或使用内部类型。
      例如，这允许你提供一个可以接受名称或数字的 JSON 字段。**
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md [hunk 11]
```diff
@@ -0,0 +1,1292 @@
...
+
+    <!-- 
+    - **updateStrategy.rollingUpdate.maxUnavailable** (IntOrString)
+
+      The maximum number of DaemonSet pods that can be unavailable during the update. Value can be an absolute number (ex: 5) or a percentage of total number of DaemonSet pods at the start of the update (ex: 10%). Absolute number is calculated from percentage by rounding up. This cannot be 0 if MaxSurge is 0 Default value is 1. Example: when this is set to 30%, at most 30% of the total number of nodes that should be running the daemon pod (i.e. status.desiredNumberScheduled) can have their pods stopped for an update at any given time. The update starts by stopping at most 30% of those DaemonSet pods and then brings up new DaemonSet pods in their place. Once the new pods are available, it then proceeds onto other DaemonSet pods, thus ensuring that at least 70% of original number of DaemonSet pods are available at all times during the update.
+    -->
+    - **updateStrategy.rollingUpdate.maxUnavailable** (IntOrString)
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md:183]: 183 前面添加空行

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md [hunk 12]
```diff
@@ -0,0 +1,1292 @@
...
+
+      更新期间不可用的 DaemonSet Pod 的最大数量。值可以是绝对数（例如：5）或更新开始时 DaemonSet Pod 总数的百分比（例如：10%）。
+      绝对数是通过四舍五入的百分比计算得出的。如果 MaxSurge 为 0，则不能为 0 默认值为 1。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md:186]:
```suggestion
      绝对数是通过四舍五入的百分比计算得出的。如果 maxSurge 为 0，则此值不能为 0。默认值为 1。
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md [hunk 13]
```diff
@@ -0,0 +1,1292 @@
...
+      例如：当设置为 30% 时，最多 30% 的应该运行守护程序 Pod 的节点总数（即 status.desiredNumberScheduled）
+      可以拥有它们的 Pod 在任何给定时间停止更新。更新首先停止最多 30% 的 DaemonSet Pod，然后在它们的位置启动新的 DaemonSet Pod。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md:188]:
```suggestion
      可以在任何给定时间停止更新。更新首先停止最多 30% 的 DaemonSet Pod，
      然后在它们所在的位置启动新的 DaemonSet Pod。
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md [hunk 14]
```diff
@@ -0,0 +1,1292 @@
...
+      一旦新的 Pod 可用，它就会继续到其他 DaemonSet Pod，从而确保在更新期间至少 70% 的原始 DaemonSet Pod 数量始终可用。
+      
+      <!-- 
+      <a name="IntOrString"></a>
+      *IntOrString is a type that can hold an int32 or a string.  When used in JSON or YAML marshalling and unmarshalling, it produces or consumes the inner type.  This allows you to have, for example, a JSON field that can accept a name or number.*
+      -->
+      <a name="IntOrString"></a>
+      **IntOrString 是一种可以保存 int32 或字符串的类型。在 JSON 或 YAML 编组和解组中使用时，它会生成或使用内部类型。例如，这允许您拥有一个可以接受名称或数字的 JSON 字段。**
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md:196]:
```suggestion
      **IntOrString 是一种可以保存 int32 或字符串的类型。在 JSON 或 YAML 编组和解组中使用时，它会生成或使用内部类型。例如，这允许你提供一个可以接受名称或数字的 JSON 字段。**
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md [hunk 15]
```diff
@@ -0,0 +1,1292 @@
...
+
+<!-- 
+- **revisionHistoryLimit** (int32)
+
+  The number of old history to retain to allow rollback. This is a pointer to distinguish between explicit zero and not specified. Defaults to 10.
+-->
+- **revisionHistoryLimit** (int32)
+
+  保留以允许回滚的旧历史记录的数量。这是个指针，用来区分明确的零和未指定的指针。默认值是 10。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md:205]:
```suggestion
  用来允许回滚而保留的旧历史记录的数量。此字段是个指针，用来区分明确的零值和未设定的指针。默认值是 10。
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md [hunk 16]
```diff
@@ -0,0 +1,1292 @@
...
+<hr>
+
+<!--
+- **numberReady** (int32), required
+
+  numberReady is the number of nodes that should be running the daemon pod and have one or more of the daemon pod running with a Ready Condition.
+-->
+- **numberReady** (int32), 必须
+
+  numberReady 是应该运行守护进程 Pod 并且有一个或多个守护进程 Pod 以就绪条件运行的节点数。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md:226]:
```suggestion
  numberReady 是应该运行守护进程 Pod 并且有一个或多个 DaemonSet Pod 以就绪状况运行的节点数。
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md [hunk 17]
```diff
@@ -0,0 +1,1292 @@
...
+
+<!--
+- **numberAvailable** (int32)
+
+  The number of nodes that should be running the daemon pod and have one or more of the daemon pod running and available (ready for at least spec.minReadySeconds)
+-->
+- **numberAvailable** (int32)
+
+  应该运行守护进程 Pod 并有一个或多个守护进程 Pod 正在运行和可用（至少准备好 spec.minReadySeconds）的节点数量。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md:235]:
```suggestion
  应该运行守护进程 Pod 并有一个或多个守护进程 Pod 正在运行和可用（就绪时长超过 spec.minReadySeconds）的节点数量。
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md [hunk 18]
```diff
@@ -0,0 +1,1292 @@
...
+  运行至少 1 个守护进程 Pod 并且应该运行守护进程 Pod 的节点数。多信息： https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/
+
+<!--
+- **updatedNumberScheduled** (int32)
+
+  The total number of nodes that are running updated daemon pod
+-->
+- **updatedNumberScheduled** (int32)
+
+  正在运行更新的守护进程的节点总数。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md:280]:
```suggestion
  正在运行更新后的守护进程 Pod 的节点总数。
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md [hunk 19]
```diff
@@ -0,0 +1,1292 @@
...
+
+  **补丁策略：根据 `type` 键合并**
+
+  <!--  
+  Represents the latest available observations of a DaemonSet's current state.
+
+  <a name="DaemonSetCondition"></a>
+  *DaemonSetCondition describes the state of a DaemonSet at a certain point.*
+  -->
+  表示 DaemonSet 当前状态的最新可用观察。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md:306]:
```suggestion
  表示 DaemonSet 当前状态的最新可用观测信息。
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md [hunk 20]
```diff
@@ -0,0 +1,1292 @@
...
+  <!--
+  - **conditions.status** (string), required
+
+    Status of the condition, one of True, False, Unknown.
+
+  - **conditions.type** (string), required
+
+    Type of DaemonSet condition.
+  -->
+  - **conditions.status** (string), 必须
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md:320]: 320 前面空行

```suggestion
  - **conditions.status** (string)，必需
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md [hunk 21]
```diff
@@ -0,0 +1,1292 @@
...
+
+    状况的状态,True、False、Unknown 之一。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md:322]:
```suggestion
    状况的状态，True、False、Unknown 之一。
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md [hunk 22]
```diff
@@ -0,0 +1,1292 @@
...
+
+  - **conditions.type** (string), 必须
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md:324]:
```suggestion
  - **conditions.type** (string)，必需
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md [hunk 23]
```diff
@@ -0,0 +1,1292 @@
...
+
+    DaemonSet 状况的类型。
+
+  <!--
+  - **conditions.lastTransitionTime** (Time)
+
+    Last time the condition transitioned from one status to another. 
+  -->
+  - **conditions.lastTransitionTime** (Time)
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md:333]: 前面添加空行

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md [hunk 24]
```diff
@@ -0,0 +1,1292 @@
...
+
+  A list of daemon sets.
+-->
+- **metadata** (<a href="{{< ref "../common-definitions/list-meta#ListMeta" >}}">ListMeta</a>)
+
+  标准列表元数据。更多信息： https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+
+- **items** ([]<a href="{{< ref "../workload-resources/daemon-set-v1#DaemonSet" >}}">DaemonSet</a>), 必须
+
+  守护进程集的列表。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md:403]:
```suggestion
  DaemonSet 的列表。
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md [hunk 25]
```diff
@@ -0,0 +1,1292 @@
...
+GET /apis/apps/v1/namespaces/{namespace}/daemonsets/{name}
+
+#### 参数
+
+<!--
+- **name** (*in path*): string, required
+
+  name of the DaemonSet
+-->
+- **name** (*in path*): string, 必须
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md:432]: 必需，不是“必需”

Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md:432]:
```suggestion
- **name** (**路径参数**)：string，必需
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md [hunk 26]
```diff
@@ -0,0 +1,1292 @@
...
+- **namespace** (*in path*): string, required
+
+  <a href="{{< ref "../common-parameters/common-parameters#namespace" >}}">namespace</a>
+
+
+- **pretty** (*in query*): string
+
+  <a href="{{< ref "../common-parameters/common-parameters#pretty" >}}">pretty</a>
+-->
+- **namespace** (*in path*): string, 必须
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md:446]:
```suggestion
- **namespace** (**路径参数**)：string，必需
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md [hunk 27]
```diff
@@ -0,0 +1,1292 @@
...
+
+  <a href="{{< ref "../common-parameters/common-parameters#namespace" >}}">namespace</a>
+
+
+- **pretty** (*in query*): string
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md:451]:
```suggestion
- **pretty** (**查询参数**)：string
```

**Review by sarazqy — COMMENTED** [review]
thanks to review again

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md [hunk 28]
```diff
@@ -0,0 +1,1598 @@
...
+    属性值可以是绝对数量（例如：5）或所需 Pod 的百分比（例如：10%）。
+    如果 maxUnavailable 为 0，则该值不能为 0。绝对数是通过四舍五入从百分比计算得出的，最小值为 1。
+    默认值为 0。示例：当设置为 30% 时，最多为节点总数的 30% 节点上应该运行守护程序 Pod（即 status.desiredNumberScheduled）
+    可以在旧 Pod 标记为已删除之前创建一个新 Pod。更新首先在 30% 的节点上启动新的 Pod。
+    一旦更新的 Pod 可用（就绪时长至少 minReadySeconds 秒），该节点上的旧 DaemonSet pod 就会被标记为已删除。
+    如果旧 Pod 因任何原因变得不可用（Ready 转换为 false、被驱逐或节点被腾空），
+    则会立即在该节点上创建更新的 Pod，而不考虑激增限制。
+    允许激增意味着如果就绪检查失败，任何给定节点上的守护程序集消耗的资源可能会翻倍，
+    因此资源密集型守护程序集应该考虑到它们可能会在中断期间导致驱逐。
+    此字段是 Beta 字段，由 DaemonSetUpdateSurge 特性门启用/禁用。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md:175]: 这一大段缩进不对。要多缩进两个空格

**Comment by tengqm** [comment]
/approve

This looks ready to merge once the dependency is resolved.

**Comment by Sea-n** [comment]
Wait for `pod-template-v1.md`.
- #34319

**Comment by Sea-n** [comment]
Hey @sarazqy,

Please avoid use `fixes #xxx` in PR description for umbrella issues.
We can use `related: #xxx` or `part of: #xxx` so that the umbrella issue would not be closed unintentionally.

**Comment by sarazqy** [comment]
> Hey @sarazqy,
> 
> Please avoid use `fixes #xxx` in PR description for umbrella issues. We can use `related: #xxx` or `part of: #xxx` so that the umbrella issue would not be closed unintentionally.

image

**Comment by sarazqy** [comment]
> Hey @sarazqy,
> 
> Please avoid use `fixes #xxx` in PR description for umbrella issues. We can use `related: #xxx` or `part of: #xxx` so that the umbrella issue would not be closed unintentionally.

----I wrote it as you suggested earlier
@Sea-n 

[Sea-n](https://github.com/Sea-n) commented [on 15 Jun](https://github.com/kubernetes/website/issues/34193#issuecomment-1155963817)
FYI, you can use Fixes #34193 in your PR description, so that the bot will automatically close this issue when it's done.

小技巧：可以在 PR 描述裡面加上一行文字寫上 Fixes #34193，這樣被 merge 時才會自動關閉相應 issue

**Comment by Sea-n** [comment]
Yeah, it's a little bit tricky.
For PR that fixes the entire issue, we would use `fixes #xxx` to close it automatically.

But this PR is part of an umbrella issue, this PR just done part of the huge job.
So we don't want the issue to be closed while others are working on part of it.

**Comment by tengqm** [comment]
Please rebase this onto 'main' branch where 'pod-v1.md' has been translated.

**Comment by tengqm** [comment]
Please rebase this onto 'main' branch where 'pod-template-v1.md' has been translated.

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md [hunk 29]
```diff
@@ -0,0 +1,1598 @@
...
+- **namespace** (*in path*): string, required
+
+  <a href="{{< ref "../common-parameters/common-parameters#namespace" >}}">namespace</a>
+
+
+- **body**: <a href="{{< ref "../workload-resources/daemon-set-v1#DaemonSet" >}}">DaemonSet</a>, required
+-->
+- **namespace** (**路径参数**): string, 必需
+
+<a href="{{< ref "../common-parameters/common-parameters#namespace" >}}">namespace</a>
```
Comment (zhangxyjlu) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md:853]:
```suggestion
  <a href="{{< ref "../common-parameters/common-parameters#namespace" >}}">namespace</a>
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md [hunk 30]
```diff
@@ -0,0 +1,1598 @@
...
+  
+  - **updateStrategy.rollingUpdate** (RollingUpdateDaemonSet)
+
+    滚动更新配置参数。仅在 type 值为 "RollingUpdate" 时出现。
+    
+    <!-- 
+    <a name="RollingUpdateDaemonSet"></a>
+    *Spec to control the desired behavior of daemon set rolling update.*
+    -->
+    <a name="RollingUpdateDaemonSet"></a>
```
Comment (zhangxyjlu) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md:154]: 可删除154行

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md [hunk 31]
```diff
@@ -0,0 +1,1598 @@
...
+      则会立即在该节点上创建更新的 Pod，而不考虑激增限制。
+      允许激增意味着如果就绪检查失败，任何给定节点上的守护程序集消耗的资源可能会翻倍，
+      因此资源密集型守护程序集应该考虑到它们可能会在中断期间导致驱逐。
+      此字段是 Beta 字段，由 DaemonSetUpdateSurge 特性门启用/禁用。
+  
+      <!-- 
+      <a name="IntOrString"></a>
+      *IntOrString is a type that can hold an int32 or a string.  When used in JSON or YAML marshalling and unmarshalling, it produces or consumes the inner type.  This allows you to have, for example, a JSON field that can accept a name or number.*
+      -->
+      <a name="IntOrString"></a>
```
Comment (zhangxyjlu) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md:181]: 可删除181行

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md [hunk 32]
```diff
@@ -0,0 +1,1598 @@
...
+      例如：当设置为 30% 时，最多节点总数 30% 的、应该运行守护程序的节点总数（即 status.desiredNumberScheduled）
+      可以在任何给定时间停止更新。更新首先停止最多 30% 的 DaemonSet Pod，
+      然后在它们的位置启动新的 DaemonSet Pod。
+      一旦新的 Pod 可用，它就会继续处理其他 DaemonSet Pod，从而确保在更新期间至少 70% 的原始 DaemonSet Pod 数量始终可用。
+      
+      <!-- 
+      <a name="IntOrString"></a>
+      *IntOrString is a type that can hold an int32 or a string.  When used in JSON or YAML marshalling and unmarshalling, it produces or consumes the inner type.  This allows you to have, for example, a JSON field that can accept a name or number.*
+      -->
+      <a name="IntOrString"></a>
```
Comment (zhangxyjlu) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md:204]: 可删除204行

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md [hunk 33]
```diff
@@ -0,0 +1,1598 @@
...
+  -->
+  
+  - **conditions.lastTransitionTime** (Time)
+
+    状况上次从一种状态转换到另一种状态的时间。
+    <!--
+    <a name="Time"></a>
+    *Time is a wrapper around time.Time which supports correct marshaling to YAML and JSON.  Wrappers are provided for many of the factory methods that the time package offers.*
+    -->
+    <a name="Time"></a>
```
Comment (zhangxyjlu) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md:351]: 可删除351行

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md [hunk 34]
```diff
@@ -0,0 +1,1598 @@
...
+<hr>
+
+<!--
+- **selector** (<a href="{{< ref "../common-definitions/label-selector#LabelSelector" >}}">LabelSelector</a>), required
+
+  A label query over pods that are managed by the daemon set. Must match in order to be controlled. It must match the pod template's labels. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors
+-->
+- **selector** (<a href="{{< ref "../common-definitions/label-selector#LabelSelector" >}}">LabelSelector</a>), 必需
+
+  对由守护程序集管理的 Pod 的标签查询。Pod 必须匹配此查询才能被此 DaemonSet 控制.查询条件必须与 Pod 模板的标签匹配。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md:92]:
```suggestion
  对由守护程序集管理的 Pod 的标签查询。Pod 必须匹配此查询才能被此 DaemonSet 控制。
  查询条件必须与 Pod 模板的标签匹配。
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md [hunk 35]
```diff
@@ -0,0 +1,1598 @@
...
+  更多信息： https://kubernetes.io/zh-cn/concepts/overview/working-with-objects/labels/#label-selectors
+
+<!--
+- **template** (<a href="{{< ref "../workload-resources/pod-template-v1#PodTemplateSpec" >}}">PodTemplateSpec</a>), required
+
+  An object that describes the pod that will be created. The DaemonSet will create exactly one copy of this pod on every node that matches the template's node selector (or on every node if no node selector is specified). More info: https://kubernetes.io/docs/concepts/workloads/controllers/replicationcontroller#pod-template
+-->
+- **template** (<a href="{{< ref "../workload-resources/pod-template-v1#PodTemplateSpec" >}}">PodTemplateSpec</a>), 必需
+
+  描述将要创建的 Pod 的对象。DaemonSet 将在与模板的节点选择器匹配的每个节点上（如果未指定节点选择器，则在每个节点上）准确创建此 Pod 的副本。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md:102]:
```suggestion
  描述将要创建的 Pod 的对象。DaemonSet 将在与模板的节点选择器匹配的每个节点上
  （如果未指定节点选择器，则在每个节点上）准确创建此 Pod 的副本。
```

**Review by tengqm — COMMENTED** [review]
/approve

**Comment by tengqm** [comment]
Preview: https://deploy-preview-34985--kubernetes-io-main-staging.netlify.app/zh-cn/docs/reference/kubernetes-api/workload-resources/daemon-set-v1/

**Pull request merged**